tests/ci/integration/tpm2_tss_patch/aws-lc-tpm2-tss.patch

```diff
@@ -1,18 +1,26 @@
-From cf4383fff44f86c03bfd19de31e73039bc204562 Mon Sep 17 00:00:00 2001
-From: Justin W Smith <103147162+justsmth@users.noreply.github.com>
-Date: Mon, 22 Jan 2024 10:20:12 -0500
-Subject: [PATCH] Minor fixes for AWS-LC
+From 0b702b55bfece6c2910ea065bf0dffa9c1e1008e Mon Sep 17 00:00:00 2001
+From: Juergen Repp <juergen_repp@web.de>
```

Comment:
This is just an artifact of how I did the rebase of the commit to create the patch. This is mostly the original patch, just rebased onto a new commit.